### PR TITLE
reproducer for gh-5559

### PR DESCRIPTION
**Pull request by jsafarik**
//test: `@gh-5559 or @gh-integrations-odata-create`

### CODE REVIEW

**Comment by jsafarik** [comment]
included test of  `integrations-odata-create` as well just to make sure the refactoring of method for creating odata credentials is ok.

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by jsafarik** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by jsafarik** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
@jsafarik it didn't run the tests you wanted, the correct syntax should be:

//test: `@gh-5559 or @gh-integrations-odata-create` - everything together in the backticks

**Comment by jsafarik** [comment]
@avano oh shoot, correcting it right now, thanks :)

**Comment by jsafarik** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: ui-tests/src/test/resources/features/integrations/odata.feature [hunk 1]
```diff
@@ -276,3 +276,13 @@ Feature: OData Connector
...
     Then check that "whatever" entity in "Users" collection contains
       | Gender | MALE |
+
+    @gh-5559
+    @reproducer
+    @odata-server-certificate
+    Scenario: OData connector needs server certificate
+      Given create OData https credentials
+      And log into the Syndesis
+      And created connections
```
Comment (avano) [inline ui-tests/src/test/resources/features/integrations/odata.feature]: when

File: ui-tests/src/test/resources/features/integrations/odata.feature [hunk 2]
```diff
@@ -276,3 +276,13 @@ Feature: OData Connector
...
+        | OData | odataHttps | ODataHttps | sample OData service |
+      Then check visibility of "Connection created Connection ODataHttps successfully created" in alert-success notification
```
Comment (avano) [inline ui-tests/src/test/resources/features/integrations/odata.feature]: add a new line

File: ui-tests/src/test/resources/features/integrations/odata.feature (hunk 1 above)
Comment (jsafarik) [reply ui-tests/src/test/resources/features/integrations/odata.feature]: changed

File: ui-tests/src/test/resources/features/integrations/odata.feature (hunk 2 above)
Comment (jsafarik) [reply ui-tests/src/test/resources/features/integrations/odata.feature]: added

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert 
Connection - CRUD | Create and delete 
Connection - CRUD | Kebab menu 
Connection - CRUD | Kebab menu edit and view 
OData Connector | OData connector needs server certificate 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by jsafarik** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert 
OData Connector | OData connector needs server certificate 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by jsafarik** [comment]
retest this please

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request merged**